gateways/adyen/adyen.go

```diff
@@ -78,12 +78,12 @@ func (client *AdyenClient) Authorize(request *sleet.AuthorizationRequest) (*slee
 		}
 	}
 
-	if result.ResultCode == adyen_common.Refused || result.ResultCode == adyen_common.Error {
+	if result.ResultCode == adyen_common.Authorised {
```

Comment:
not sure if it's worth, but we can write this without the else case, `if !result.ResultCode == adyen_common.Authorised`

Reply:
Yeah actually I hate if/else let me fix it